test/core/formattersTests.ts

```diff
@@ -1,193 +1,280 @@
-    it("shows exactly [precision] digits", () => {
+  describe("fixed()", () => {
+    it("shows correct amount of digits according to precision", () => {
       let fixed3 = Plottable.Formatters.fixed();
-      let result = fixed3(1);
-      assert.strictEqual(result, "1.000", "defaults to three decimal places");
-      result = fixed3(1.234);
-      assert.strictEqual(result, "1.234", "shows three decimal places");
-      result = fixed3(1.2346);
-      assert.strictEqual(result, "1.235", "changed values are not shown (get turned into empty strings)");
-    });
+      assert.strictEqual(fixed3(1), "1.000", "shows three decimal places");
+      assert.strictEqual(fixed3(-1), "-1.000", "shows three decimal places");
+      assert.strictEqual(fixed3(1.234), "1.234", "shows three decimal places");
+      assert.strictEqual(fixed3(-1.234), "-1.234", "shows three decimal places");
+      assert.strictEqual(fixed3(1.2346111111), "1.235", "shows three decimal places");
+      assert.strictEqual(fixed3(-1.2346111111), "-1.235", "shows three decimal places");
+
+      assert.strictEqual(fixed3(Infinity), "Infinity", "formats non-numeric number correctly");
+      assert.strictEqual(fixed3(-Infinity), "-Infinity", "formats non-numeric number correctly");
+      assert.strictEqual(fixed3(NaN), "NaN", "formats non-numeric number correctly");
 
-    it("precision can be changed", () => {
       let fixed2 = Plottable.Formatters.fixed(2);
-      let result = fixed2(1);
-      assert.strictEqual(result, "1.00", "formatter was changed to show only two decimal places");
+      assert.strictEqual(fixed2(1), "1.00", "shows two decimal places");
+      assert.strictEqual(fixed2(-1), "-1.00", "shows two decimal places");
+      assert.strictEqual(fixed2(1.23), "1.23", "shows two decimal places");
+      assert.strictEqual(fixed2(-1.23), "-1.23", "shows two decimal places");
+      assert.strictEqual(fixed2(1.2346111111), "1.23", "shows two decimal places");
+      assert.strictEqual(fixed2(-1.2346111111), "-1.23", "shows two decimal places");
+
+      let fixed0 = Plottable.Formatters.fixed(0);
+      assert.strictEqual(fixed0(1), "1", "shows no decimal places");
+      assert.strictEqual(fixed0(-1), "-1", "shows no decimal places");
+      assert.strictEqual(fixed0(1.23), "1", "shows no decimal places");
+      assert.strictEqual(fixed0(-1.23), "-1", "shows no decimal places");
     });
 
-    it("can be set to show rounded values", () => {
-      let fixed3 = Plottable.Formatters.fixed(3);
-      let result = fixed3(1.2349);
-      assert.strictEqual(result, "1.235", "long values are rounded correctly");
+    it.skip("throws exception for non-numeric values", () => {
+      let nonNumbricValues = [null, undefined, "123", "abc", ""];
+      let fixed = Plottable.Formatters.fixed();
+      nonNumbricValues.forEach((value) =>
+        (<any> assert).throws(() => fixed(<any> value), Error,
+          "", `${value} is not a valid value for Formatter.fixed`)
```

Comment:
Makes sense. In that case add a random string there. If the pattern match will fail you will remember to update it upon unskipping, otherwise you won't notice.